src/components/input-field/examples/input-field-textarea.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../style/internal/code-styles.scss';
+
+p {
+    font-size: small;
+}
+
+limel-input-field[type='textarea'] {
+    --textarea-height: 150px;
```

Comment:
I think I disagree with this… 🤔

This means we're overriding the default look of the component in the example that's there to show what the component looks like. I'm not sure that makes a lot of sense.

Reply:
Ah, this has just been moved…

Well, I still disagree with it being there in the first place then 😆

Reply:
:zap: